CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 4.7.2 (Unreleased)
+#### Notes
+Added the capability to handle OneView Appliance SNMP Settings
+
+#### Features supported with current release:
+- Appliance Device Read Community
+- Appliance Device SNMP v1 Trap Destinations
+- Appliance Device SNMP v3 Trap Destinations
+- Appliance Device SNMP v3 Users
+
 # 4.7.1
```

Comment:
just noticed that 4.7.1 was never released. So lets merge your changes and 4.7.1 change and mark it as 4.8 as per semantic versioning

Reply:
Changed in commit c86234875cc51d63d58d4577bac70ada15d4c995